bin/checkver.ps1

```diff
@@ -214,6 +220,15 @@ while ($in_progress -gt 0) {
         }
     }
 
+    if ($xpath) {
+        # Getting version from XML, using XPath
+        $ver = ([xml]$page).SelectSingleNode($xpath).'#text'
+        if (!$ver) {
+            next "couldn't find '$xpath' in $url"
+            continue
+        }
+    }
+
     if ($jsonpath -and $regexp) {
```

Comment:
Would be nice to implement concurent usage of `xpath` and `regex`.

(Example: https://github.com/lukesampson/scoop-extras/blob/ec81f5723b3d53bdd63b42b203bf41537607925f/bucket/eclipse-testing.json)

There you would need `xpath` to get version and `regex` to strip that `/R` from it.

Reply:
I don't quite understand the use-case you provided, do you mean to add the follow codes?

```powershell
    if ($xpath -and $regexp) {
        $page = $ver
        $ver = ''
    }
```

Reply:
Yes. Because then flow will be next: 

1. `xpath` will be used to get version.
1. `regex` (could look like `(.*)/R`) will be used
1. Version will be applied to first group of regex

But version needs to strip that `/R`